### PR TITLE
Feat/adding active users aggregates dim config

### PR DESCRIPTION
**Pull request by kik-kik**
# Feat/adding active users aggregates dim config

### CODE REVIEW

**Comment by kik-kik** [comment]
WIP until Lucia is back and we can confirm if there is anything else we should be watching out for.

**Review by gkabbz — APPROVED** [review]
Looks like we have tests for both active_users_aggregates view as well as the active_users_aggregates_v1 table. Do we need both? Wouldn't that kick off two separate alerts for the same issue?

**Comment by kik-kik** [comment]
@gkabbz good point, not sure why both were there. I removed the config for the table directly.

File: dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          - Windows_95
+          - Windows_98
+          - Windows_NT
+          - OpenBSD
+          - SunOS
+          - HubertOS
+    - type: numeric_value_matches
+      params:
+        columns:
+          - first_seen_year: "> 2020"
```
Comment (kik-kik) [inline dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml:98]: @lucia-vargas-a is this a correct assumption? It seems we have a lot of values which would be less that that.

File: dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    - type: combined_column_uniqueness
+      params:
+        columns:
+          - submission_date  # probably don't need this since we run checks on a single partition at a time
```
[the remark this answers is not in the document]
Comment (lucia-vargas-a) [reply dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml:107]: No, there are several values for these same combination.

File: dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          - casual_user
+          - regular_user
+          - infrequent_user
+          - other
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - app_name
+        expected_values:
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml:71]: Some missing values? See [the full list here](https://github.com/mozilla/dim/blob/2fd107e4349d8fc70ce0538a905cd2ac9430a188/dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml#L74).

**Review by lucia-vargas-a — CHANGES_REQUESTED** [review]
Missing values and comments in files.

File: dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kik-kik) [reply dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml:107]: What combination should give us a unique entry?

[the remark this answers is not in the document]
Comment (kik-kik) [reply dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml]: @lucia-vargas-a

**Pull request merged**